src/Microsoft.AspNet.Server.Kestrel/Http/Connection.cs

```diff
@@ -29,15 +30,17 @@ public Connection(ListenerContext context, UvStreamHandle socket) : base(context
             ConnectionControl = this;
 
             _connectionId = Interlocked.Increment(ref _lastConnectionId);
+
+            SocketInput = new SocketInput(Memory2);
+            SocketOutput = new SocketOutput(Thread, _socket, _connectionId, Log);
+            _frame = new Frame(this);
         }
 
         public void Start()
         {
             Log.ConnectionStart(_connectionId);
 
-            SocketInput = new SocketInput(Memory);
-            SocketOutput = new SocketOutput(Thread, _socket, _connectionId, Log);
-            _frame = new Frame(this);
+            Task.Run((Action)_frame.ProcessFraming);
```

Comment:
Path is:
[Task.Run](https://github.com/dotnet/coreclr/blob/4cf8a6b082d9bb1789facd996d8265d3908757b2/src/mscorlib/src/System/Threading/Tasks/Task.cs#L5659)
* ```StackCrawlMark stackMark = StackCrawlMark.LookForMyCaller```
* [TaskFactory.StartNew](https://github.com/dotnet/coreclr/blob/4cf8a6b082d9bb1789facd996d8265d3908757b2/src/mscorlib/src/System/Threading/Tasks/TaskFactory.cs#L569)
* InternalStartNew
 * ```new Task```
 * ```Task.PossiblyCaptureContext```
 * ```Task.ScheduleAndStart```
  * ...
  * [```TaskScheduler.InternalQueueTask```](https://github.com/dotnet/coreclr/blob/4cf8a6b082d9bb1789facd996d8265d3908757b2/src/mscorlib/src/System/Threading/Tasks/TaskScheduler.cs#L265)
    * [```TaskScheduler.QueueTask```](https://github.com/dotnet/coreclr/blob/4cf8a6b082d9bb1789facd996d8265d3908757b2/src/mscorlib/src/System/Threading/Tasks/TaskScheduler.cs#L632)
    * [SynchronizationContext.Post](https://github.com/dotnet/coreclr/blob/4cf8a6b082d9bb1789facd996d8265d3908757b2/src/mscorlib/src/System/Threading/SynchronizationContext.cs#L126)
      * ThreadPool.QueueUserWorkItem